src/libraries/System.Collections.Immutable/tests/Frozen/FrozenSetAlternateLookupTests.cs

```diff
@@ -0,0 +1,140 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.Frozen.Tests
+{
+    public class FrozenSetAlternateLookupTests
+    {
+        [Fact]
+        public void AlternateLookup_Empty()
+        {
+            Assert.False(FrozenSet<string>.Empty.TryGetAlternateLookup<ReadOnlySpan<char>>(out _));
+
+            foreach (StringComparer comparer in new[] { StringComparer.Ordinal, StringComparer.OrdinalIgnoreCase })
```

Comment:
Nit: this could have been a theory in a test with a different name.

Reply:
Revised the test.